lib/SoyOptions.js

```diff
@@ -21,6 +21,18 @@ function SoyOptions() {
   this.inputDir = process.cwd()
 
   /**
+   * Directory containing the Closure Templates SoyToJsSrcCompiler.jar
+   * @type {string}
+   */
+  this.soyJarDir = process.cwd()
```

Comment:
(not seeing a way to require arguments on the SoyOptions side, but the command will fail out quickly if an invalid path is specified)